### DISCUSSION TITLE
Deploy changes bug

### DISCUSSION THREAD

**Discussion by lucasfelipedonascimento** [planetscale/discussion#212] [Bug Reports] [answered]
Hello people. 
I've been trying to deploy a database since yesterday, and now nothing.
![Captura de tela de 2022-06-06 11-40-30](https://user-images.githubusercontent.com/62124129/172192358-daa8a3db-37c0-4d89-af90-49303a984721.png)

Have any of you also had this same bug?

**Comment by mscoutermarsh** [comment]
@lucasfelipedonascimento We're looking into it now!

**Comment by mscoutermarsh** [comment, marked as answer]
@lucasfelipedonascimento Should be in a good shape now, we just fixed the issue.

**Reply by lucasfelipedonascimento** [reply]
Thank you.
